### DISCUSSION TITLE
Unable to decrypt attribute "ssl"

### DISCUSSION THREAD

**Discussion by NareeratPr** [Security-Onion-Solutions/securityonion#11421] [2.4] [answered]
### Version

2.4.10

### Installation Method

Network installation on Ubuntu

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Does not meet minimum requirements

### CPU

4

### RAM

16GB

### Storage for /

118G

### Storage for /nsm

491G

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I installed Security Onion 2.4 on Ubuntu 22.04 and found the error **"statusCode":500,"error":"Internal Server Error","message":"Unable to decrypt attribute \"ssl\"** from Elastic Fleet during the installation.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/124574170/4c3fc29f-401e-44d9-843c-814aa310f169)

Another issue is my security onion cannot ingest any events from the monitoring interface.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/124574170/0c803b9f-0282-4a82-b1e6-88439a0290a5)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/124574170/ba4d12f1-9905-4427-b316-0c35e5c8a406)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
If you chose Standalone, then you probably need more RAM:
https://docs.securityonion.net/en/2.4/hardware.html#standalone-deployments

**Reply by NareeratPr** [reply]
Does that mean if I increase more RAM, this issue will be fixed, right?
Could you recommend the installation type if I want to monitor and detect suspicious traffic from an interface (mirror port from core switch)?

**Reply by dougburks** [reply, marked as answer]
> Does that mean if I increase more RAM, this issue will be fixed, right?

I can't make any guarantees, but you'll have a better chance of success if you increase to [at least 24GB RAM ](https://docs.securityonion.net/en/2.4/hardware.html#standalone-deployments) and perform a fresh installation using our Security Onion ISO image:
https://docs.securityonion.net/en/2.4/best-practices.html
https://docs.securityonion.net/en/2.4/download.html#download
https://docs.securityonion.net/en/2.4/os.html
https://docs.securityonion.net/en/2.4/installation.html

> Could you recommend the installation type if I want to monitor and detect suspicious traffic from an interface (mirror port from core switch)?

Standalone may work depending on how much traffic you're monitoring and how much future expansion you want to plan for:
https://docs.securityonion.net/en/2.4/architecture.html#standalone

Long term you may want to consider moving to a distributed deployment as it provides many advantages:
https://docs.securityonion.net/en/2.4/architecture.html#distributed
